src/bpftrace.cpp

```diff
@@ -119,10 +119,12 @@ int BPFtrace::add_probe(const ast::AttachPoint &ap,
   auto probe = generate_probe(ap, p, usdt_location_idx);
 
   // Add the new probe(s) to resources
-  if (ap.provider == "BEGIN" || ap.provider == "END") {
+  if (ap.provider == "BEGIN" || ap.provider == "END" ||
+      ap.provider == "bpftrace") {
     // special probes
     probe.path = "/proc/self/exe";
-    probe.attach_point = ap.provider + "_trigger";
+    auto target = ap.target.empty() ? "" : "_" + ap.target;
+    probe.attach_point = ap.provider + target + "_trigger";
```

Comment:
Not valid ATM but once we support multiple signals, we'll need to add the `_<func>` part to the attach point name, right?